packages/react-interactive-tab/stories/InteractiveTab/index.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import { Meta } from '@storybook/react';
+import { InteractiveTab } from '@fluentui-contrib/react-interactive-tab';
+export { Default } from './Default.stories';
+
+const meta: Meta<typeof InteractiveTab> = {
```

Comment:
I know, copy paste from other projects in this repo. It would be great to start following community standards ( which you already started in core - which is great ! )

```
const meta = {
 ....
} satisfies Meta<typeof InteractiveTab>
```